test/inductor/test_mkldnn_pattern_matcher.py

```diff
@@ -1069,6 +1069,7 @@ def test_qconv2d_xpu(self):
     @skipIfNoDynamoSupport
     @skipIfNoONEDNNBF16
     @skipIfNoONEDNN
+    @skipIfRocm
```

Comment:
Shouldn't we only skip for MI300?

Reply:
only gfx94* (MI300) support fp8
https://github.com/pytorch/pytorch/blob/a70191da41abf98807f212e1bbb8daeca9936ec1/torch/testing/_internal/common_cuda.py#L92

Reply:
But this is a new skip right? that means it should already be running fine

Reply:
What happens when you run this on MI200? does the UT gracefully return saying "not supported"?

Reply:
on MI210
```
root@sc-hw-smc-acc-02:/var/lib/jenkins/pytorch/test# python inductor/test_mkldnn_pattern_matcher.py -v -k test_qconv2d_int8_mixed_bf16

  test_qconv2d_int8_mixed_bf16 (__main__.TestPatternMatcher)
This testcase will quantize a single Conv2d module with int8_mixed_bf16 quantization. ... skip: Quantized operations require BF16 support. (0.124s)

OK (skipped=1)
```

Reply:
Resolving this conversation, but we need to look at enabling this unit test on both MI200 and MI300, if applicable